tests/python/unittest/test_gluon.py

```diff
@@ -132,7 +132,7 @@ def test_symbol_block():
     out = smodel(mx.sym.var('in'))
     assert len(out) == len(outputs.list_outputs())
 
-    class Net(nn.HybridBlock):
```

Comment:
Is this going to break? A lot of code is written like this. We can't break this

Reply:
I think having duplicate documentation is more tolerable than breaking codes.

Reply:
Is there a way to hide it?

Reply:
I recall that there's `:exclude-members:` function in sphinx that can do it. Let me look into it.

Reply:
Got it. See the latest update.